frontend/src/Pages/ComplaintOverview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+import Card from "../components/Card";
+import { AxiosComplaintApi } from "../utils/AxiosApis";
+
+interface iCard {
+  _id: string;
+  priority: number;
+  title: string;
+  created: string;
+  description: string;
+  reporter: string;
+}
+
+const CardOverview = () => {
+  const [comps, setComps] = useState([]);
+
+  useEffect(() => {
+    async function fetchComplaints() {
+      const result = await AxiosComplaintApi.get("complaints");
+      setComps(result.data);
```

Comment:
You have to parse it first.
```javascript
const result = await AxiosComplaintApi.get("complaints");
const data = await result.json()
```

Reply:
my understanding was that axios does this automatically for us and this is one of the benefits vs fetch

Reply:
After verifying; you're absolutely right 👍